wiki/Tournaments/UCUP/2021/en.md

```diff
@@ -0,0 +1,559 @@
+---
+tags:
+  - UCUP
+  - UCUP 4
+  - UCUP 2021
```

Comment:
```suggestion
  - UCUP4
  - UCUP 2021
  - UCUP2021
```

Reply:
you kinda removed `UCUP 4`

Reply:
aaaaaaaaaaaa

ok it should be good now hopefully D: